thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/completeness/checker/DataCompletenessScheduler.java

```diff
@@ -24,18 +24,14 @@
   private DataCompletenessJobRunner dataCompletenessJobRunner;
   private DataCompletenessJobContext dataCompletenessJobContext;
   private List<String> datasetsToCheck = new ArrayList<>();
-  private static final ThirdEyeCacheRegistry CACHE_REGISTRY = ThirdEyeCacheRegistry.getInstance();
 
   public DataCompletenessScheduler(String datasetsToCheck) {
     scheduledExecutorService = Executors.newSingleThreadScheduledExecutor();
     if (StringUtils.isNotBlank(datasetsToCheck)) {
       this.datasetsToCheck = Lists.newArrayList(datasetsToCheck.split(","));
-    } else {
-      this.datasetsToCheck = CACHE_REGISTRY.getCollectionsCache().getCollections();
```

Comment:
Is this line supposed to be removed? I just want to make sure we want the data checker does no-op when the datasets to check are not provided.